examples/svelte/basic/src/App.svelte

```diff
@@ -104,7 +104,7 @@
     }))
   }
 
-  const instance = createTableInstance(table, options)
+  const instance = createTableInstance(table, $options)
```

Comment:
In this case, it should be the store (`options`) that is passed to `createTableInstance` rather than its value (`$options`), because that way the adapter can internally call `subscribe` on the store to listen to changes.

Reply:
ah ok, wasn't clear on that. interesting